js/navigation/NavigationLayout.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react-navigation';
```

Comment:
React can be imported from 'react' not 'react-navigation'. Please change that asap. Also if you are not using it, please remove it.